Sources/DataPipeline/DataPipeline.swift

```diff
@@ -0,0 +1,121 @@
+import CioInternalCommon
+
+public class DataPipeline: ModuleTopLevelObject<CustomerIOInstance>, CustomerIOInstance {
+    @Atomic public private(set) static var shared = DataPipeline()
+    @Atomic public private(set) static var moduleConfig: DataPipelineConfigOptions!
+    private static let moduleName = "DataPipeline"
+
+    private init() {
+        super.init(moduleName: Self.moduleName)
+    }
+
+    /**
+     Initialize the shared `instance` of `DataPipeline`.
+     Call this function when your app launches, before using `DataPipeline.shared`.
```

Comment:
> Call this function when your app launches, before using `DataPipeline.shared`.

This comment makes me think you're suggesting that customers should be calling `DataPipeline.initialize` instead of `CustomerIO.initialize` to initialize the SDK and use `DataPipeline.shared.identify()` instead of `CustomerIO.shared.identify()`. 

Maybe I am reading this comment wrong and it's not suggesting that customers call this function on app start but instead we need to when `CustomerIO.initialize` is called?